server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/DefaultMetaRepository.java

```diff
@@ -172,18 +202,16 @@ private static List<MirrorCredential> loadCredentials(Map<String, Entry<?>> entr
     }
 
     @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
-    @JsonSubTypes({
-            @Type(value = SingleMirrorConfig.class, name = "single"),
-            @Type(value = MultipleMirrorConfig.class, name = "multiple")
-    })
+    @JsonSubTypes(@Type(value = SingleMirrorConfig.class, name = "single"))
     private abstract static class MirrorConfig {
```

Comment:
Will combine `MirrorConfig` and `SingleMirrorConfig` together after @ikhoon's work for mirroring UI is done.

Reply:
Combine them in this PR instead of waiting for @ikhoon's work. 😉